packages/edit-navigation/src/components/menu-locations-editor/index.js

```diff
@@ -1,3 +1,117 @@
+/**
+ * External dependencies
+ */
+import { forEach, find, pick } from 'lodash';
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import {
+	Button,
+	Panel,
+	PanelBody,
+	SelectControl,
+	Spinner,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const MenuSelectControl = ( { onChange, menus, menusList, location } ) => {
+	const [ menuId, setMenuId ] = useState( false );
+
+	return (
+		<>
+			{ menus && (
```

Comment:
We can remove this check and have `MenuSelectControl` require that `menus` is an array.

Reply:
found this in unneeded actually